server/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -2341,18 +2339,6 @@ public ActionRequestValidationException validate(
                 validationException = rescorer.validate(this, validationException);
             }
         }
-
-        if (pointInTimeBuilder() == null && sorts() != null) {
```

Comment:
I think that maybe we should also remove `90_search_after#_shard_doc sort` yaml test as well?